### PR TITLE
Liveness Probe and Readiness Probe with Defaults

### PR DESCRIPTION
**Pull request by hjwalt**
For issue #105 . @regadas can I get your review? This is my first time contributing with golang (usually coding with Java).

### CODE REVIEW

**Review by regadas — APPROVED** [review]
@hjwalt Thanks for this!

File: controllers/flinkcluster_converter.go [hunk 1]
```diff
@@ -14,6 +14,7 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
+// TODO FIND error causes
```
Comment (regadas) [inline controllers/flinkcluster_converter.go:17]: @hjwalt what's this about?

Comment (hjwalt) [reply controllers/flinkcluster_converter.go]: I missed removing that todo. It is the probe configuration tests. 

When updating the tests, I thought that the defaults will also be set by the converter if it is not currently set, turns out thats not the case. Whether or not we keep this behaviour is a separate discussion, so I didn't do a change on those points.

Comment (regadas) [reply controllers/flinkcluster_converter.go]: Gotcha! Yeah defaults are only set through the webhook

**Review by regadas — APPROVED** [review]
Thanks again!

**Comment by hjwalt** [comment]
Thank you for supporting the project @regadas !

**Pull request merged**